docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeSuiteLifecycleController.java

```diff
@@ -45,13 +53,15 @@
     private Instance<DockerClientExecutor> dockerClientExecutor;
 
     public void startAutoContainers(@Observes(precedence = 100) BeforeSuite event, CubeConfiguration cubeConfiguration, CubeDockerConfiguration dockerConfiguration) {
+        beforeAutoStart.fire(new BeforeAutoStart());
```

Comment:
I would rename all the `Event<...>` to have an `Event` suffix.

I think it's more readable when you see

```
beforeAutoStartEvent.fire(new BeforeAutoStart());
```

WDYT?